Promote Announced to OUFR (#8147)
OfficeDev/office-ui-fabric-react: packages/office-ui-fabric-react/src/components/Announced/docs/AnnouncedOverview.md

The Announced component aims to fill several of the accessibility gaps that exist in various web application experiences.
It provides text for the screen reader in certain scenarios that are lacking comprehensive updates, particularly those showing
the completion status or progress of operation(s).

Some real-world applications of the component include copying, uploading, or moving many files, deleting or renaming a single file,
"lazy loading" of page sections that do not appear all at once, and appearance of search results.

The Announced component currently has the following 4 documented use cases:

> 1. [Quick Actions](#examples/announced/quickactions): Small operations like editing text, or deletion that are short enough that they do not require a status during progress.

Reviewer: ```suggestion
1. [Quick Actions](#examples/announced/quickactions): Operations such as editing text or deletion that are short enough that they do not require a status during progress.
```

> 2. [Search Results](#examples/announced/searchresults): Appearance of search like results such as in contact fields, or search boxes.

Reviewer: ```suggestion
2. [Search Results](#examples/announced/searchresults): Appearance of search results such as in contact fields or search boxes.
```
